pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -35,18 +35,28 @@ type Controller struct {
 	machinery.ProjectNameMixin
 
 	ControllerRuntimeVersion string
+	IsLegacyLayout           bool
 }
 
 // SetTemplateDefaults implements file.Template
 func (f *Controller) SetTemplateDefaults() error {
 	if f.Path == "" {
 		if f.MultiGroup && f.Resource.Group != "" {
-			f.Path = filepath.Join("controllers", "%[group]", "%[kind]_controller.go")
+			if f.IsLegacyLayout {
+				f.Path = filepath.Join("controllers", "%[group]", "%[kind]_controller.go")
+			} else {
+				f.Path = filepath.Join("pkg", "controllers", "%[group]", "%[kind]_controller.go")
+			}
 		} else {
-			f.Path = filepath.Join("controllers", "%[kind]_controller.go")
+			if f.IsLegacyLayout {
+				f.Path = filepath.Join("controllers", "%[kind]_controller.go")
+			} else {
+				f.Path = filepath.Join("pkg", "controllers", "%[kind]_controller.go")
+			}
 		}
 	}
 	f.Path = f.Resource.Replacer().Replace(f.Path)
+	fmt.Println(f.Path)
```

Comment:
Debugging?

Reply:
No. That is to ensure the default standard of printing the path when we do the scaffold as done in the other actions. See; https://github.com/kubernetes-sigs/kubebuilder/search?q=fmt.Println%28f.Path%29